### PR TITLE
[Impeller] Use blit for root pass copy

### PR DESCRIPTION
**Pull request by bdero**
Use a blit for the root pass copy when available in the backend.

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
RSLGTM

File: impeller/renderer/renderer_unittests.cc [hunk 1]
```diff
@@ -514,7 +514,7 @@ TEST_P(RendererTest, CanBlitTextureToTexture) {
       // Blit `bridge` to the top left corner of the texture.
       pass->AddCopy(bridge, texture);
 
-      pass->EncodeCommands(context->GetResourceAllocator());
+      assert(pass->EncodeCommands(context->GetResourceAllocator()));
```
Comment (chinmaygarde) [inline impeller/renderer/renderer_unittests.cc:517]: `assert`s get stripped away in !defined(NDEBUG). Besides, let's not terminate the process on this error. Return false like elsewhere?

**Review by chinmaygarde — CHANGES_REQUESTED** [review]

**Comment by chinmaygarde** [comment]
Really supportive of this change except for the stripped away thing that you want to do :)

File: impeller/renderer/renderer_unittests.cc (hunk 1 above)
Comment (bdero) [reply impeller/renderer/renderer_unittests.cc]: Whoops, fixed.

**Review by chinmaygarde — APPROVED** [review]

**Pull request merged**